src/content/docs/en/upgrade-astro.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: Upgrade Astro
+description: Learn how to upgrade Astro
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import Version from '~/components/Version.astro';
+
+
+This guide covers how to update your version of Astro and related dependencies, how to learn what has changed from one version to the next, and how to understand Astro's versioning system and corresponding documentation updates.
+
+## What has changed?
+The latest release of Astro is <Version pkgName="astro" />.
+
+You can find an exhaustive list of all changes in [Astro's changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md), and important instructions for upgrading to each new [major version](#major-changes) in our [upgrade guides](#upgrade-guides).
```

Comment:
Perhaps this is suggestion needs it's own separate discussion (to ensure consistency throughout the docs). Thoughts on making the external link to Astro's Changelog open in a new tab? 

Since markdown doesn't have a built in way to do this, a simple component could be created to implement this. While this is a subjective suggestion, personally, I do find it a bit annoying to have the link open in the current page.

Reply:
Our practice is to never open links in new tabs for consistency and accessibility. It's generally considered better practice to allow users to control that behaviour (everyone can choose to open in a new tab, but if you force a new tab, you take that control away from them).

Reply:
Good to know! Thank you for clarifying this :)!